docs/specs/servers/blockchain/blockchain-server-api.md

```diff
@@ -276,3 +276,141 @@ The POST request body should be in JSON format with the following structure:
 
 * `400 Bad Request` - some parameters in request body were missed or wrong.
 * `401 Unauthorized` - projectID verification error.
+
+## Convertation
```

Comment:
Think it's "Conversion"

Reply:
Yes. Seems like there is no word in English as "converation", instead "conversation" could be used which refers to the act or process of converting something from one form, function, or purpose to another

Reply:
Haha, Grammarly shows me that conversation and convertation are both exists. 
We need help from a native speaker ) @chris13524 help us please )

Reply:
Jumping in here to save you guys from this debate 😂  it's `Conversion` (as Rocky noted) when speaking about conversion of assets, e.g. "conversion rate" when speaking about exchanging currencies

Reply:
Thanks @bkrem 😅 I'll make changes to the name!

Reply:
Lool, my bad, yeah it's totally `conversion`, not `conversation`, thanks @bkrem , haha

Reply:
`Convert tokens`

Reply:
Fixed in https://github.com/WalletConnect/walletconnect-specs/pull/207/commits/93e69f8f22d9c39e829205f3bc0b6609480d3a7d